t/plugin/jwt-auth.t

```diff
@@ -519,3 +519,23 @@ GET /t
 \{"error_msg":"failed to check the configuration of plugin jwt-auth err: additional properties forbidden, found key"\}
 --- no_error_log
 [error]
+
+
+
+=== TEST 25: get the schema by schema_type
+--- request
+GET /apisix/admin/schema/plugins/jwt-auth?schema_type=consumer
+--- response_body
+{"required":["key"],"properties":{"exp":{"minimum":1,"type":"integer"},"algorithm":{"type":"string","default":"HS256","enum":["HS256","HS512","RS256"]},"base64_secret":{"default":false,"type":"boolean"},"secret":{"type":"string"},"key":{"type":"string"}},"additionalProperties":false,"type":"object"}
+--- no_error_log
+[error]
+
+
+
+=== TEST 26: get the schema by error schema_type
```

Comment:
The name of test case should be `by the default schema_type`.

Reply:
ok, here is the test case when passing schema_type to obtain consumer_schema, I added the test case obtained by default schema_type

Reply:
@spacewander done.